### PR TITLE
convert members list to json and add sponsors page

### LINKED ISSUE #160: Add CarGurus as Sponsors on the website
We need a Sponsors section to display CarGurus as the Working Group sponsor. 
Please find the logo attached.
CarGurus

The Sponsors can be listed right after the Members list. @shanda WDYT?

Thanks,
Zahra

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [x] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by sxa — APPROVED** [review]
LGTM. It may also be worth - as a courtesy so people can find it - adding a link to this at the bottom of the sponsors page with something along the lines of "In addition, the project has sponsors that provide financial donations to the project but are not part of the working group"

**Comment by sxa** [comment]
NOTE: Also fixes https://github.com/adoptium/adoptium.net/issues/160

File: src/js/members.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      case 'enterprise':
+        enterpriseMembers.push(member)
+      break;
+      case 'participant':
+        participantMembers.push(member)
+      break;
+    }
+  }
+
+  // Randomly mix up members logos
```
Comment (tellison) [inline src/js/members.js:23]: Not sure how I feel about a page that changes each time it is loaded.

**Review by tellison — APPROVED** [review]

File: src/js/members.js (hunk 1 above)
Comment (gdams) [reply src/js/members.js:23]: I think it's the only way to fairly show each member on our members page?

**Pull request merged**